gradle/libs.versions.toml

```diff
@@ -22,6 +22,7 @@ dokka = "1.6.10"
 mkdocs = "2.2.0"
 mkdocs-plugin-awesome-pages = "2.6.1"
 node = "16.13.2"
+bignum = "0.3.4"
```

Comment:
This is left-over from the previous iteration. 😊

Reply:
Thanks for noticing, I missed that one when cleaning up 👍